constants/constants.go

```diff
@@ -135,6 +135,12 @@ const RepeatFlag = "repetitions"
 //ShortRepeatFlag - shorthand flag for repetitions
 const ShortRepeatFlag = "rep"
 
+//VersionFlag defines version of seed spec to use
+const VersionFlag = "version"
```

Comment:
I don't see any evidence of this const being used. Is it being used in consuming project?

Reply:
This is for specifying the seed spec version in seed-cli commands

Reply:
Good point, I initially thought there'd be shared constants but it's just been a big pain to have them in a separate project